Users/login.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express")
+const router = express.Router()
+
+const {User} = require("./user.model.js")
+
+router.route("/")
+  .post(async(req,res) => {
+      const {email, password} = req.body;
+
+    try{
+        const findUser = await User.find({email: email})
```

Comment:
`findOne()` is a better alternative as find() is about to be deprecated

Reply:
Done